lib/class-sp-wp-search.php

```diff
@@ -305,6 +305,17 @@ public static function wp_to_es_args( $args ) {
 						);
 
 						break;
+
+					case 'author':
+						$es_query_args['aggregations'][ $label ] = array(
+							'terms' => array(
+								'field' => $facet['field'],
```

Comment:
From my perspective, it should be static. You need to either choose the login, nicename, or the user id. For the taxonomies we are using `slug`, so `post_author.login` would probably be the most consistent with that.